paimon-core/src/main/java/org/apache/paimon/iceberg/IcebergCommitCallback.java

```diff
@@ -1230,6 +1328,147 @@ private List<IcebergManifestFileMeta> createDvManifestFileMetas(Snapshot snapsho
+        long totalFilesSize = Math.max(0, metrics.totalFilesSize);
+        long totalDeleteFiles = Math.max(0, metrics.totalDeleteFiles);
+        long totalPositionDeletes = Math.max(0, metrics.totalPositionDeletes);
+        long totalEqualityDeletes = Math.max(0, metrics.totalEqualityDeletes);
+
+        summary.put("added-data-files", Long.toString(addedDataFiles));
+        summary.put("added-records", Long.toString(addedRecords));
+        summary.put("added-files-size", Long.toString(addedFilesSize));
+        summary.put("deleted-data-files", Long.toString(deletedDataFiles));
+        summary.put("deleted-records", Long.toString(deletedRecords));
+        summary.put("deleted-files-size", Long.toString(deletedFilesSize));
+        summary.put("changed-partition-count", Long.toString(changedPartitionCount));
+        summary.put("total-records", Long.toString(totalRecords));
+        summary.put("total-data-files", Long.toString(totalDataFiles));
+        summary.put("total-files-size", Long.toString(totalFilesSize));
+        summary.put("total-delete-files", Long.toString(totalDeleteFiles));
+        summary.put("total-position-deletes", Long.toString(totalPositionDeletes));
+        summary.put("total-equality-deletes", Long.toString(totalEqualityDeletes));
+
+        Map<String, String> properties = snapshot.properties();
+        if (properties != null) {
+            properties.forEach(
+                    (key, value) -> {
+                        if (value != null) {
+                            summary.put(key, value);
+                        }
+                    });
+        }
+
+        return summary;
+    }
+
+    private long computeLiveDataFileCount(List<IcebergManifestFileMeta> manifestMetas) {
+        return manifestMetas.stream()
+                .mapToLong(
+                        meta ->
+                                meta.addedFilesCount()
+                                        + meta.existingFilesCount()
+                                        - meta.deletedFilesCount())
+                .sum();
+    }
+
+    private long computeLiveRowCount(List<IcebergManifestFileMeta> manifestMetas) {
+        return manifestMetas.stream()
+                .mapToLong(
+                        meta ->
+                                meta.addedRowsCount()
+                                        + meta.existingRowsCount()
+                                        - meta.deletedRowsCount())
+                .sum();
+    }
+
+    private long computeLiveDeleteFileCount(List<IcebergManifestFileMeta> manifestMetas) {
+        return manifestMetas.stream()
+                .mapToLong(
+                        meta ->
+                                meta.addedFilesCount()
+                                        + meta.existingFilesCount()
+                                        - meta.deletedFilesCount())
+                .sum();
+    }
+
+    private long computeDeleteRowCount(List<IcebergManifestFileMeta> manifestMetas) {
```

Comment:
computeLiveRowCount is the same with computeDeleteRowCount now, please check them.